core/src/main/java/com/google/errorprone/bugpatterns/StatementSwitchToExpressionSwitch.java

```diff
@@ -790,6 +906,32 @@ private static int findBlockStatementIndex(TreePath treePath, BlockTree blockTre
     return -1;
   }
 
+  private static boolean precedingTwoStatementsMultiVariableDeclarations(
+      List<StatementTree> precedingStatements, VisitorState state) {
+    if (precedingStatements.size() >= 2) {
+
+      if (precedingStatements.get(precedingStatements.size() - 1) instanceof VariableTree
+          && precedingStatements.get(precedingStatements.size() - 2) instanceof VariableTree) {
+
+        VariableTree variableTree1 =
+            (VariableTree) precedingStatements.get(precedingStatements.size() - 1);
+        VariableTree variableTree2 =
+            (VariableTree) precedingStatements.get(precedingStatements.size() - 2);
```

Comment:
```suggestion
      if (precedingStatements.get(precedingStatements.size() - 1) instanceof VariableTree variableTree1
          && precedingStatements.get(precedingStatements.size() - 2) instanceof VariableTree variableTree2) {
```

(A similar simplification is possible below.)